### ISSUE TITLE
How to continue training after a break?

### ISSUE THREAD

**Issue by gaopeng-eugene** [DrSleep/tensorflow-deeplab-lfov#6]
For example, the training process stoped after 20K iterations. How to continue training from 20K iterations?

**Comment by DrSleep** [comment]
You would need to run train.py --restore_from with path to your ckpt file. Note also that as default, the train.py script saves only trainable variables (thus no running mean/variance are saved). You would need to modify it to save all global variables (tf.global_variables())

**Comment by gaopeng-eugene** [comment]
How to restore learning rate for Adam optimiser?

**Comment by DrSleep** [comment]
IIRC saving tf.global_variables() and then restoring should take care of that.
